app/helpers/application_helper.rb

```diff
@@ -288,34 +282,30 @@ def flot_bar_chart(name, xaxis_label, yaxis_label, data, options = {})
                 }.merge(options))
   end
 
-  def action_buttons(*args)
-    toolbar_action_buttons args
-  end
-
   def select_action_button(title, options = {}, *args)
     # the no-buttons code is needed for users with less permissions
-    return unless args
```

Comment:
This check is useless, `args` defaults to `[] ` even if no arguments are passed.